packages/app-defaults/src/defaults/apis.ts

```diff
@@ -296,4 +300,14 @@ export const apis = [
       });
     },
   }),
+  createApiFactory({
```

Comment:
For discussion: since the permission-backend isn't included in create-app yet, should we set up the permissionApi in packages/app rather than app-defaults?

Reply:
To keep this PR internally consistent, I've added a changeset for this for now in 1e49c23eb7. Can pull it out along with this change if we end up moving this setup to packages/app until the permission-backend lands in create-app.